src/simplify_algebra.cpp

```diff
@@ -851,6 +851,84 @@ struct find_div_const
     }
 };
 
+struct find_zero_div_const
+{
+    auto matcher() const { return match::name("div")(match::arg(1)(match::has_value(0.0f))); }
+
+    [[noreturn]] void apply([[maybe_unused]] module& m,
+                            [[maybe_unused]] const match::matcher_result& r) const
```

Comment:
For unused parameters you can just remove the parameter name: `void apply(module&, const match::matcher_result&)` instead of using `[[maybe_unused]]`.

Reply:
Okay, I wasn't sure which was better/our desired standard but I can change this. Shouldn't be an issue as long as we have an understanding of what's clear to the reader.